DFE-Digital/early-careers-framework
# Start induction term listing changes

## Ticket and context

Ticket: 
[246](https://dfedigital.atlassian.net/jira/software/projects/CST/boards/119?assignee=5f3543bb3e9e2e004db3189d&selectedIssue=CST-246) 
[247](https://dfedigital.atlassian.net/jira/software/projects/CST/boards/119?assignee=5f3543bb3e9e2e004db3189d&selectedIssue=CST-247)

246
<img width="741" alt="Screenshot 2022-01-06 at 08 15 56" src="https://user-images.githubusercontent.com/69353998/148351153-23034a81-06a0-4450-9d4a-c088bab452f2.png">

247
<img width="1091" alt="Screenshot 2022-01-06 at 08 15 11" src="https://user-images.githubusercontent.com/69353998/148351076-45451221-3ed0-445b-96c9-6e03f17d6ce4.png">


## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Login in as `cpd-test+tutor-10@example.com`
3. go to the dashboard. All users who haven't been assigned a start date will be grouped as Autumn 2021. 
4. Add an ECT or Mentor, follow the journey and Autumn 2021 should be removed as an option. 
5. Complete the journey and that ppt should be on your dashboard with the term you selected.
